tests/test_openmxparser.py

```diff
@@ -217,6 +233,7 @@ def test_C2N2(parser):
     assert method.electronic.smearing.width == approx(K_to_J(500))
     assert method.dft.xc_functional.exchange[0].name == 'LDA_X'
     assert method.dft.xc_functional.correlation[0].name == 'LDA_C_PZ'
+    assert (method.k_mesh.grid == [1, 1, 1]).all()
```

Comment:
This case is the only one which exercises the case with only one (implicit) gamma point. Can we please also check the multiplicities here?